tools/oecert/host/host.cpp

```diff
@@ -553,11 +552,11 @@ static oe_result_t _process_params(oe_enclave_t* enclave)
     oe_result_t result = OE_FAILURE;
 
 #if defined(__linux__)
-    char* sgx_aesm_env = getenv(SGX_AESM_ADDR);
+    char* sgx_asem_env = getenv(SGX_AESM_ADDR);
```

Comment:
This typo correction need not be reverted, but if it's easiest to revert collectively, it's ok.